odata-data-aggregation-ext/3.3 Transformations Producing a Subset.md

```diff
@@ -237,7 +237,7 @@ results in
 
 The `skip` transformation takes a non-negative integer $c$ as argument. Let $A$ be a copy of the input set with a total order that extends any existing order of the input set but is otherwise chosen by the service. The total order MUST be stable across requests.
 
-The transformation excludes from the output set the first $c$ instances of $A$. It keeps all remaining instances in the same order as they occur in $A$.
+The transformation excludes from the output set the first $c$ instances in $A$ (possibly with repetitions). It keeps all remaining instances in the same order as they occur in $A$.
```

Comment:
It is about occurrences, not instances. Therefore apply "occurrence" like at other places, e.g compute

Proposal: "The transformation excludes from the output set the first $c$ occurrences in $A$."